packages/aleo-gateway/src/raw_signature.rs

```diff
@@ -2,12 +2,29 @@ use error_stack::Report;
 
 use crate::{AleoValue, Error};
 
+/*
+sign1tg9nwza05k89vyspuemc8r5vkz54f3v3gl4s6x05atqtgwnyyvq8ccamwql5txztuevplttghu5eyprlmth6ysgvsz9mgzjslf8guq5r22qjwn4zc0pzv87twjygsz9m7ekljmuw4jpzf68rwuq99r0tp735vs6220q7tp60nr7llkwstcvu49wdhydx5x2s3sftjskzawhqvnvcgml
+*/
+
 // TODO: nonce is skipped
 #[derive(Clone, Debug)]
 pub struct RawSignature {
     pub signature: Vec<u8>,
 }
 
+impl RawSignature {
+    pub fn new(sig: &str) -> Self {
+        let signature = sig.as_bytes().to_vec();
+        RawSignature { signature }
+    }
+}
+
+impl Default for RawSignature {
+    fn default() -> Self {
+        RawSignature::new("sign1tg9nwza05k89vyspuemc8r5vkz54f3v3gl4s6x05atqtgwnyyvq8ccamwql5txztuevplttghu5eyprlmth6ysgvsz9mgzjslf8guq5r22qjwn4zc0pzv87twjygsz9m7ekljmuw4jpzf68rwuq99r0tp735vs6220q7tp60nr7llkwstcvu49wdhydx5x2s3sftjskzawhqvnvcgml")
```

Comment:
Would it be helpful to keep this as a `static`, so other parts of the code can reference it?